14/umbraco-cms/reference/security/external-login-providers.md

```diff
@@ -544,12 +626,12 @@ public static class ProviderBackofficeAuthenticationExtensions
                 {
                     // The scheme must be set with this method to work for the back office
                     var schemeName =
-                        backOfficeAuthenticationBuilder.SchemeForBackOffice(GoogleBackOfficeExternalLoginProviderOptions
+                        backOfficeAuthenticationBuilder.SchemeForBackOffice(GenericOfficeExternalLoginProviderOptions
                             .SchemeName);
 
                     ArgumentNullException.ThrowIfNull(schemeName);
 
-                    backOfficeAuthenticationBuilder.AddProvider(
+                    backOfficeAuthenticationBuilder.AddGenericProvider(
```

Comment:
getting the following error on AddGenericProvider, is it the same for you?

![image](https://github.com/umbraco/UmbracoDocs/assets/83591955/be482fb3-6688-4e18-849f-c96e7c6dcf5a)